.github/workflows/service_test_http.yml

```diff
@@ -48,7 +49,7 @@ jobs:
         run: |
           mkdir /tmp/static
           cp -r tests/data/* /tmp/static
-          nginx -c `pwd`/src/services/http/fixtures/nginx.conf
+          nginx -c ../fixtures/http/nginx.conf
```

Comment:
nignx already use it's binary path as the base of path, so please prefix `pwd` here